### PR TITLE
Hazmat suits incoming!

### PR DESCRIPTION
**Pull request by Koboboldic**
Added four stylish hazmat suits, gloves and oxygen backpack for them.
Added a new gas mask in your collection.
Now oxygen tanks will be full at roundstart.
Added a new labcoat.

dreammaker_2018-10-16_00-57-57
dreamseeker_2018-10-15_18-27-20
dreammaker_2018-10-15_22-43-07

### CODE REVIEW

**Comment by JakeDaBoss** [comment]
Did you have specific ideas in mind as to how these would be utilized?

**Comment by Koboboldic** [comment]
Yep. Yellow hazmat suit will be used in... for example in 049 cell. And other suits like white or cyan in medical/virology or research. These suits is great fits in this place, with that lore.
Also old SS13 bio suits looks awful.

File: html/changelogs/FurryDog-woof.yml [hunk 1]
```diff
@@ -33,4 +33,6 @@ delete-after: True
 # Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
 # Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
 changes: 
-  - rscadd: "Added a log message that will appear for you and everyone who has you in their sight if you examine something"
```
Comment (JakeDaBoss) [inline html/changelogs/FurryDog-woof.yml]: You might want to check this out. It apparently removed something?

Comment (Koboboldic) [reply html/changelogs/FurryDog-woof.yml]: This is changelog .-.

Comment (Koboboldic) [reply html/changelogs/FurryDog-woof.yml]: Oh

Comment (TheSleepyCatmom) [reply html/changelogs/FurryDog-woof.yml]: Pretty sure it's removing the old suits. Also, HAZMAT suits are ONLY a yellow. You won't find white or grey hazmat suits ever. Or blue, or anything. They're meant to be ambiguous, and are only colored in a military setting, to black. At least in most places.

Hazmat Suits are Class A suits, class B suits would be for chemical work, class C suits are for less volatile things where only a gas mask and a sterile suit is required (IE laboratories). Might be relevant to check that in the future :^)

Comment (Koboboldic) [reply html/changelogs/FurryDog-woof.yml]: I don't think so.

**Pull request merged**